src/topology/running.rs

```diff
@@ -453,9 +461,15 @@ impl RunningTopology {
 
         // Broadcast changes to subscribers.
         if !self.watch.0.is_closed() {
+            // TODO handle sending ids
```

Comment:
If we're not going to do this as part of the PR then it's probably a good idea to open an issue so we don't forget.

Reply:
I implemented what was needed 😉